src/hooks/apis/useDeleteInstance.js

```diff
@@ -4,7 +4,7 @@ import apiRequest from '../../services/apiRequest';
 
 const deleteInstance = async (instanceID) => {
   const { data } = await apiRequest.delete(
-    `/api/rhacs/v1/centrals/${instanceID}`
+    `/api/rhacs/v1/centrals/${instanceID}?async=true`
```

Comment:
Curious to what this does. Does the API immediately return an HTTP response and then delete the instance in the background, instead of blocking the HTTP response until the instance has been fully deleted?

Reply:
Yes, that is correct 💯